keras/utils/test_utils.py

```diff
@@ -36,7 +36,7 @@ def get_test_data(nb_train=1000, nb_test=500, input_shape=(10,),
 
 
 def layer_test(layer_cls, kwargs={}, input_shape=None, input_dtype=None,
-               input_data=None, expected_output=None, expected_output_dtype=None):
+               input_data=None, expected_output=None, expected_output_dtype=None, with_None=True):
```

Comment:
yeah, I was thinking about that... but the problem is deconv in TF doesn't handle `None` in the shape dim. the test has to specify all dimensions. so this `None` refers to the number of samples
how should we call it instead?